### PR TITLE
Update mdatp-6-systemconfigurationprofiles.png

### PR DESCRIPTION
**Pull request by msbemba**
Change in the screenshot as deployment channel is available now

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/7095

File: microsoft-365/security/defender-endpoint/mac-install-with-intune.md [hunk 1]
```diff
@@ -107,6 +107,7 @@ This profile contains a license information for Microsoft Defender for Endpoint,
     > ![Custom Configuration Profile - name.](images/mdatp-6-systemconfigurationprofiles-2.png)
 
 1. Choose a name for the configuration profile name, e.g., "Defender for Endpoint onboarding for macOS".
+1. Choose a Deployment channel.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/mac-install-with-intune.md]:
```suggestion
1. Choose a deployment channel.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by jerryabo** [comment]
Thanks, I think we want to also work in/breadcrumb the What's New guidance/details here as well please @msbemba https://docs.microsoft.com/en-us/mem/intune/fundamentals/whats-new#new-deployment-channel-setting-for-custom-device-configuration-profiles-on-macos-devices

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@Dansimp Please merge this PR. Thanks

**Comment by msbemba** [comment]
issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/7194

**Pull request merged**